### ISSUE TITLE
Can I use part of the built-in environment? 

### ISSUE THREAD

**Issue by BrianN92** [Microsoft/AutonomousDrivingCookbook#33]
Hi all, I am doing the E2E deep learning in Autonomous Cooking book. I like the "City" environment because of the urban scenes and some moving cars and people. But it is too large and can not be run smoothly on my PC. So I am wondering if I can use part of it, like fewer streets, buildings or cars, so that it can be run better? Thank you!

**Comment by mitchellspryn** [comment]
**tl;dr**: No.

Longer answer: The city environment was developed by the [AirSim](https://github.com/Microsoft/AirSim) team. In order to pare down the map, you'd have to open it up in the unreal editor, delete a bunch of the assets / portions of the map, and rebuild the binaries. The problem is that many of the assets were developed in-house and have not been open sourced (and are unlikely to be in the near future due to licensing agreements), so there's no way for you to load the project into unreal.

There are a few potential workarounds:
* Upgrade your hardware.
* Try the neighborhood environment. This is still "city-like," and much simpler. It may suit your needs.
* Spin up a GPU-enabled machine in Azure. I haven't tried it, but I'd believe that the NV machines should be capable of running that environment.
* Try windowed mode (with the -WINDOWED command line argument) or changing the renderer used (e.g. -opengl3).

Also, keep in mind that if you're trying to do the E2E tutorial on the city environment, you'll have to generate the dataset fresh; the provided dataset is for the neighborhood environment. You may also consider [opening an issue on the AirSim repo](https://github.com/Microsoft/AirSim/issues) to ask them to provide a city binary with fewer assets or to build the binary supporting [level streaming](https://forums.epicgames.com/udk/udk-development/level-design-and-creation/270486-how-can-i-use-level-streaming-with-huge-maps).
